libs/utils/utils.js

```diff
@@ -829,12 +829,16 @@ async function loadMartech({ persEnabled = false, persManifests = [] } = {}) {
 }
 
 async function checkForPageMods() {
+  const search = new URLSearchParams(window.location.search);
+  const offFlag = (val) => search.get(val) === 'off';
+  if (offFlag('mep')) {
+    document.body.dataset.mep = 'nopzn|nopzn';
```

Comment:
I see this is used again in an identical way later in the file. If another such use-case arises, it might be worth having a small method for it to be re-used everywhere